Learning/Test1_function/section-count.py

```diff
@@ -0,0 +1,30 @@
+word_meaning_dict = {
```

Comment:
use a separate file for this input instead of copying everywhere

Reply:
will do it in next PR & all will update all these changes everywhere